api/controllers/restaurantController.js

```diff
@@ -28,13 +48,28 @@ const getRestaurants = async (req, res) => {
 };
 
 const getRestaurant = async (req, res) => {
-  const { restaurantId } = req.params;
-  const restaurant = await Restaurant.findByPk(restaurantId, { include: ['cuisines', 'reviews'] });
-  if (restaurant) {
-    res.send(restaurant);
-  } else {
-    res.send(mockRestaurants[0]);
-  }
+  const { yelpId } = req.params;
```

Comment:
This should remain restaurantId if you are planning to hit the yelp endpoint. Create a yelp controller if you are hitting the yelp endpoint because restaurantId is a param that refers to the database id. This will break the put and delete requests that rely on restaurantId. yelpId also exists on the model so if you kept the restaurantId, then you could still make the request to yelp.